street_level_crime/input_helpers.py

```diff
@@ -1,14 +1,84 @@
-import re
-
-uk_postcode_format = re.compile("([Gg][Ii][Rr] 0[Aa]{2})|((([A-Za-z][0-9]{1,2})|(([A-Za-z][A-Ha-hJ-Yj-y][0-9]{1,2})|(([A-Za-z][0-9][A-Za-z])|([A-Za-z][A-Ha-hJ-Yj-y][0-9][A-Za-z]?))))\s?[0-9][A-Za-z]{2})")
-
-def validate_postcode(postcode):
-    return bool(uk_postcode_format.match(postcode))
-
-def ui_get_postcode():
-    while True:
-        postcode = input("Please, enter postcode (e.g. BH12 2ED)")
-        if validate_postcode(postcode):
-            print ("Correct postcode")
-        else:
-            print("Please, check your postcode. It must be a valid UK format (e.g. GH12 2ED)")
+import re
+import os
+
+
+def validate_postcode(postcode):
+    uk_postcode_format = re.compile("([Gg][Ii][Rr] 0[Aa]{2})|((([A-Za-z][0-9]{1,2})|(([A-Za-z][A-Ha-hJ-Yj-y][0-9]{1,2})|(([A-Za-z][0-9][A-Za-z])|([A-Za-z][A-Ha-hJ-Yj-y][0-9][A-Za-z]?))))\s?[0-9][A-Za-z]{2})")
+    return bool(uk_postcode_format.match(postcode))
+
+def validate_file_name(file_name):
+    valid_file_name = re.compile('^[^\\\<>:"/|?]+$')
+    return bool(valid_file_name.match(file_name))
+
+def directory_exists(directory):
+    if os.path.isdir(directory):
+        return True
+    else:
+        return False

```

Comment:
isdir returns a bool so we don't need the if statement
```suggestion
    return os.path.isdir(directory)
```